### PR TITLE
Append Change Log For Supporting Bookmark/Acronyms/Teamschatmessage

### PR DESCRIPTION
**Pull request by ZChristine**

### CODE REVIEW

**Comment by ZChristine** [comment]
could you help review the pull request? it's just a simple request which insert the change log. @Lauragra

File: changelog/Microsoft.MicrosoftSearch.json [hunk 1]
```diff
@@ -573,6 +573,40 @@
       "CreatedDateTime": "2022-07-14T04:11:04.628237Z",
       "WorkloadArea": "Search",
       "SubArea": ""
+    },
+    {
```
Comment (Lauragra) [inline changelog/Microsoft.MicrosoftSearch.json:577]: Are you going to update the documentation as described in the changelog entries? Generally the changelog is accompanied by updates to the docs as described.

File: changelog/Microsoft.MicrosoftSearch.json [hunk 2]
```diff
@@ -573,6 +573,40 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **acronym** member to the **entityType** enumeration.",
+          "Target": "entityType"
+        },
+        {
+          "Id": "95e7245e-ce25-4944-9f4b-d1ab9b932e69",
+          "ApiChange": "Member",
+          "ChangedApiName": "bookmark",
+          "ChangeType": "Addition",
+          "Description": "Added the [bookmark](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
```
Comment (Lauragra) [inline changelog/Microsoft.MicrosoftSearch.json:592]: You are missing the link here, does the resource topic exist? Please make sure that the sentence ends in a period.

File: changelog/Microsoft.MicrosoftSearch.json (hunk 1 above)
Comment (ZChristine) [reply changelog/Microsoft.MicrosoftSearch.json:577]: yes. the related docs for chatMessage has been updated and we will update the bookmark and acronym later.

File: changelog/Microsoft.MicrosoftSearch.json (hunk 2 above)
Comment (ZChristine) [reply changelog/Microsoft.MicrosoftSearch.json]: updated.

File: changelog/Microsoft.MicrosoftSearch.json [hunk 3]
```diff
@@ -573,6 +573,40 @@
...
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "95e7245e-ce25-4944-9f4b-d1ab9b932e69",
+          "ApiChange": "Member",
+          "ChangedApiName": "acronym",
+          "ChangeType": "Addition",
+          "Description": "Added the **acronym** member to the **entityType** enumeration.",
```
Comment (Lauragra) [inline changelog/Microsoft.MicrosoftSearch.json:584]: Please update the list of values in for **entityType** in the enums.md file and also update any resource topics that list the possible values for properties of type **entityType**.

Comment (ZChristine) [reply changelog/Microsoft.MicrosoftSearch.json:584]: > Please update the list of values in for **entityType** in the enums.md file and also update any resource topics that list the possible values for properties of type **entityType**. For example, this topic: https://learn.microsoft.com/en-us/graph/api/resources/searchrequest?view=graph-rest-beta.

Thank you for reminding. I have updated the https://learn.microsoft.com/en-us/graph/api/resources/searchrequest?view=graph-rest-beta which lists possible values for **entityType**. For other resource like api-reference/beta/resources/search-api-overview.md will be updated in another pull request https://github.com/microsoftgraph/microsoft-graph-docs/pull/18107/files#diff-82124a8c0c82be7bfcdb68e839f881957e866dfaf8694402d2a3a79151bcdf3f

**Review by Lauragra — APPROVED** [review]

**Pull request merged**